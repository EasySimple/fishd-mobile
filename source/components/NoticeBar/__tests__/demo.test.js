import webDemoTest from '../../../tests/demoTest';

webDemoTest('NoticeBar');
