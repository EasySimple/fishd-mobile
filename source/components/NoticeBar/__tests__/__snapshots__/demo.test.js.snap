// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders NoticeBar demo correctly 1`] = `
<div>
  <div
    class="components-noticebar-demo-basic"
  >
    <div
      class="demo-title"
    >
      基本用法
    </div>
    <div
      class="fm-notice-bar"
      role="alert"
      style="margin-bottom:10px"
    >
      <div
        aria-hidden="true"
        class="fm-notice-bar-icon"
      >
        <i
          class="icon iconfont fm-icon fm-icon-audio"
          style="font-size:12px"
        />
      </div>
      <div
        class="fm-notice-bar-content"
      >
        <div
          class="fm-notice-bar-marquee-wrap "
          role="marquee"
          style="overflow:hidden"
        >
          <div
            class="fm-notice-bar-marquee"
            style="position:relative;right:0;white-space:nowrap;display:inline-block"
          >
            真正的创新是没有原型可以参考的
          </div>
        </div>
      </div>
    </div>
    <div
      class="fm-notice-bar"
      role="alert"
    >
      <div
        class="fm-notice-bar-content"
      >
        <div
          class="fm-notice-bar-marquee-wrap "
          role="marquee"
          style="overflow:hidden"
        >
          <div
            class="fm-notice-bar-marquee"
            style="position:relative;right:0;white-space:nowrap;display:inline-block"
          >
            真正的创新是没有原型可以参考的
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-noticebar-demo-basic"
  >
    <div
      class="demo-title"
    >
      支持关闭
    </div>
    <div
      class="fm-notice-bar"
      role="alert"
      style="margin-bottom:10px"
    >
      <div
        aria-hidden="true"
        class="fm-notice-bar-icon"
      >
        <i
          class="icon iconfont fm-icon fm-icon-audio"
          style="font-size:12px"
        />
      </div>
      <div
        class="fm-notice-bar-content"
      >
        <div
          class="fm-notice-bar-marquee-wrap "
          role="marquee"
          style="overflow:hidden"
        >
          <div
            class="fm-notice-bar-marquee"
            style="position:relative;right:0;white-space:nowrap;display:inline-block"
          >
            真正的创新是没有原型可以参考的
          </div>
        </div>
      </div>
      <div
        aria-label="close"
        class="fm-notice-bar-operation"
        role="button"
      >
        <i
          class="icon iconfont fm-icon fm-icon-cross"
          style="font-size:12px"
        />
      </div>
    </div>
    <div
      class="fm-notice-bar"
      role="alert"
    >
      <div
        aria-hidden="true"
        class="fm-notice-bar-icon"
      >
        <i
          class="icon iconfont fm-icon fm-icon-audio"
          style="font-size:12px"
        />
      </div>
      <div
        class="fm-notice-bar-content"
      >
        <div
          class="fm-notice-bar-marquee-wrap "
          role="marquee"
          style="overflow:hidden"
        >
          <div
            class="fm-notice-bar-marquee"
            style="position:relative;right:0;white-space:nowrap;display:inline-block"
          >
            真正的创新是没有原型可以参考的
          </div>
        </div>
      </div>
      <div
        aria-label="close"
        class="fm-notice-bar-operation"
        role="button"
      >
        <span
          marqueeprops="[object Object]"
          style="color:#a1a1a1"
        >
          不再提醒
        </span>
      </div>
    </div>
  </div>
  <div
    class="components-noticebar-demo-basic"
  >
    <div
      class="demo-title"
    >
      滚动播放
    </div>
    <div
      class="fm-notice-bar"
      role="alert"
      style="margin-bottom:10px"
    >
      <div
        aria-hidden="true"
        class="fm-notice-bar-icon"
      >
        <i
          class="icon iconfont fm-icon fm-icon-audio"
          style="font-size:12px"
        />
      </div>
      <div
        class="fm-notice-bar-content"
      >
        <div
          class="fm-notice-bar-marquee-wrap "
          role="marquee"
          style="overflow:hidden"
        >
          <div
            class="fm-notice-bar-marquee"
            style="position:relative;right:0;white-space:nowrap;display:inline-block;padding:0 7.5px"
          >
            不同盯着空白页太久，勇敢的跨出第一步，其他的自然而然就发生
          </div>
        </div>
      </div>
    </div>
    <div
      class="fm-notice-bar"
      role="alert"
    >
      <div
        class="fm-notice-bar-content"
      >
        <div
          class="fm-notice-bar-marquee-wrap "
          role="marquee"
          style="overflow:hidden"
        >
          <div
            class="fm-notice-bar-marquee"
            style="position:relative;right:0;white-space:nowrap;display:inline-block;padding:0 7.5px"
          >
            不同盯着空白页太久，勇敢的跨出第一步，其他的自然而然就发生
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-noticebar-demo-basic"
  >
    <div
      class="demo-title"
    >
      查看更多
    </div>
    <div
      class="fm-notice-bar"
      role="alert"
      style="margin-bottom:10px"
    >
      <div
        aria-hidden="true"
        class="fm-notice-bar-icon"
      >
        <i
          class="icon iconfont fm-icon fm-icon-audio"
          style="font-size:12px"
        />
      </div>
      <div
        class="fm-notice-bar-content"
      >
        <div
          class="fm-notice-bar-marquee-wrap "
          role="marquee"
          style="overflow:hidden"
        >
          <div
            class="fm-notice-bar-marquee"
            style="position:relative;right:0;white-space:nowrap;display:inline-block"
          >
            真正的创新是没有原型可以参考的
          </div>
        </div>
      </div>
      <div
        aria-label="go to detail"
        class="fm-notice-bar-operation"
        role="button"
      >
        <i
          class="icon iconfont fm-icon fm-icon-allow-right"
          style="font-size:12px"
        />
      </div>
    </div>
    <div
      class="fm-notice-bar"
      role="alert"
    >
      <div
        aria-hidden="true"
        class="fm-notice-bar-icon"
      >
        <i
          class="icon iconfont fm-icon fm-icon-audio"
          style="font-size:12px"
        />
      </div>
      <div
        class="fm-notice-bar-content"
      >
        <div
          class="fm-notice-bar-marquee-wrap "
          role="marquee"
          style="overflow:hidden"
        >
          <div
            class="fm-notice-bar-marquee"
            style="position:relative;right:0;white-space:nowrap;display:inline-block;padding:0 7.5px"
          >
            真正的创新是没有原型可以参考的
          </div>
        </div>
      </div>
      <div
        aria-label="go to detail"
        class="fm-notice-bar-operation"
        role="button"
      >
        <span>
          查看更多
        </span>
      </div>
    </div>
  </div>
</div>
`;
